maze_transformer/training/training.py

```diff
@@ -37,7 +37,6 @@ def get_dataloader(
     dataset: MazeDataset, cfg: ConfigHolder, logger: WandbLogger
 ) -> DataLoader:
     length_stats: StatCounter = StatCounter(dataset.get_all_lengths())
-    logger.summary({"dataset_seq_len_stats_summary": length_stats.summary()})
     logger.summary(
```

Comment:
```suggestion
    logger.progress(
```


I reckon we should move it from the summary to a console log, because it clutters up the summary 

Side-note, I'm kind of thinking the WandbLogger interface is a bit obscure. Maybe the `progress` function should be called something different?

Reply:
Ah yeah I didn't catch the meaning of `progress`. `console`?

Reply:
I think we should update to a general W&B class that you pass a logger into, rather than a logger class that does other W&B operations like uploading artifacts. I can put up a draft if that sounds good to you

Reply:
Like 

```
W&BClass:
 def init(logger: CustomLogger):
   # etc
 def upload_dataset:
   # artifact = upload_dataset operation
   logger.wandb.log_artifact(artifact)
    
   

``

Reply:
Hmm I need to think more about this

Reply:
Does the 

```
        logging.basicConfig(
            stream=sys.stdout,
            level=logging.INFO,
            format="%(asctime)s %(levelname)s %(message)s",
            datefmt="%Y-%m-%d %H:%M:%S",
        )
```

code affect the W&B logging in any way? Maybe we could extract out the regular logger to another file and then call `logger` or `wandb` with its own logging encapsulated as necessary.

Reply:
That makes sense, happy to do that. Thanks for looking into it!

Reply:
I've moved it to the utils module for now instead of its own setup class to avoid over-engineering